ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousPlanner.java

```diff
@@ -81,6 +83,8 @@ public enum PlanningMode
    private boolean active;
    private double previousContinuousHikingSwingTime = 0.0;
 
+   private List<SideDependentList<Pose3D>> walkToGoalWayPointList = new ArrayList<>();
+
    float xRandomMargin = 0.2f;
    float nominalStanceWidth = 0.22f;
```

Comment:
isn't this a property of the robot and out of scope for this class?

Reply:
I'm not sure if its a property of the robot, I agree it shouldn't be in this class, will have to think about where this should live. Some how I'm looking for a way to tell ContinuousHiking to move on from one goal waypoint and onto the next one if it exists. That allows for clicking multiple goals like waypoints and going to all of them, then it will be easy to make a patrol feature.

Reply:
I'm planning on moving this up a class, or some how creating a remote continuous hiking manager (not really sure yet) to handle things like this. Like what ever class started the process may want to keep track of "waypoints" and then ContinuousHiking just handles the immediate goal, for example if I want to add a Patrol feature, somehow I have to remember the start and end, and go back and forth. A higher level manager could track the start and goal, and check the distance, and when ever something new needs to be done, pass that down to the ContinuousHiking Module, open to ideas about this